### DISCUSSION TITLE
Can we use Soketi ?

### DISCUSSION THREAD

**Discussion by sharathdn1** [munafio/chatify#206] [General]
Soketi WebSockets server implements Pusher v7 protocol and open-source self-hosted alternative for Pusher.

**Comment by munafio** [comment]
@sharathdn1 Currently, There are no options in Chatify out of the box to do so .. but since anything in Chatify can be customized as you need, then you can implement it yourself by removing pusher implementation and then use your own web socket server or an alternative to Pusher.

**Comment by munafio** [comment]
@sharathdn1 Now it is supported and you can use it with any other WS server, check out the [documentation](https://chatify.munafio.com/custom-websocket-server) .
